.ci/bump-version.sh

```diff
@@ -14,10 +14,12 @@ docker run --rm -t \
   --user $UID \
   -e LOCAL_USER_ID=$UID \
   -e HOME=/tmp \
+  -e AGENT_VERSION="${NEW_AGENT_VERSION}" \
   -w /app \
   -v "$(pwd):/app" \
   ruby:${RUBY_VERSION} /bin/sh -c "set -x
     gem install bundler
+    gem install elastic-apm -v ${AGENT_VERSION}
     bundle update elastic-apm"
```

Comment:
```suggestion
    bundle update"
```

We could do so to test with latest deps?

Reply:
IIUC, `bundle update elastic-apm` will update all the transitive dependencies for the elastic-apm while `bundle update` will update all of them, right?

If so, I'd prefer to be more cautious, and put that particular bump process in place with some kind of dependabot or similar ones. 

WDYT?

Reply:
Fine with me 👍